### ISSUE TITLE
pdflatex doesn't end and cannot kill the process after a fatal errors

### ISSUE THREAD

**Issue by jlperla** [thomasjo/atom-latex#461]
I am using `atom-latex` on Windows 10 with MikTeX 2.9 and Use DiCy set to true.

On fatal errors of pdflatex, the process will not end and the `Kill` doesn't do anything.

Replicating is easy for one of the errors:
1. Generate a PDF file from any latex
2. Open the PDF in a separate PDF viewer outside of atom
3. Try to recompile, which should fail because it can't overwrite the PDF

It you do this, it will get stuck.  Kill doesn't work, etc.  You have to restart Atom.

The last lines in the log show that an error occurs, as it should:
```
C:/Users/Jesse Perla/.julia/v0.6/DiffEqOperators/docs/linear_operators_overview
.tex:61:  ==> Fatal error occurred, no output PDF file produced!
```
The Message in the `LaTeX Log` says
```
[LaTex] Error: Command failed: `pdflatex -file-line-error -interaction=batchmode -recorder -synctex=1 -shell-restricted linear_operators_overview.tex`
```
The one thing that stands out here is the `-interaction=batchmode`.  When I have run pdflatex, I usually put `-interation=nonstopmode`?  Not sure if that is what is causing the problem... if so, is there a way to add that?

**Comment by jlperla** [comment]
It looks like latexmk has a `-f` command that is equivalent to the `-interaction=nonstop`... http://mirror.its.dal.ca/ctan/support/latexmk/latexmk.pdf  

Is this something you can do with your DiCy integration?

**Comment by yitzchak** [comment]
`batchmode` is just a quieter version of `nonstopmode`. I think this might have be fixed by #441 since I reworked the kill command.  Let me see if I can reproduce the issue though. What PDF viewer are you using?

**Comment by jlperla** [comment]
Thanks so much for looking at this.    The pdf viewer I use in atom is https://github.com/izuzak/atom-pdf-view  But to easily reproduce the error you need to load the pdf in a separate PDF viewer such as Adobe acrobat.

Just to be clear, though, it is not the pdf which is the issue but rather getting stuck after fatal errors... It might be all fatal errors.
